api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
+   <td><a href="http://dotnet.github.io/api/System.IO.Packaging.html">System.IO.Packaging</a></td>
+   <td>
+   Contains types that support storage of multiple data objects in a single container.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.IO.Pipes.html">System.IO.Pipes</a></td>
+   <td>
+   Contains types for interprocess communication through anonymous and/or named pipes.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Linq.html">System.Linq</a></td>
+   <td>
+   Contains types that support queries that use Language-Integrated Query (LINQ).
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Linq.Expressions.html">System.Linq.Expressions</a></td>
+   <td>
+   Contains types that enable language-level code expressions to be represented as objects in the form of expression trees.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.html">System.Net</a></td>
+   <td>
+   Provides a simple programming interface for many of the protocols used on networks today. The <a href="http://dotnet.github.io/api/System.Net.WebRequest.html">WebRequest</a> and <a href="http://dotnet.github.io/api/System.Net.WebResponse.html">WebResponse</a> classes form the basis of what are called pluggable protocols, an implementation of network services that enables you to develop applications that use Internet resources without worrying about the specific details of the individual protocols.  
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.Http.html">System.Net.Http</a></td>
+   <td>
+   Provides a programming interface for modern HTTP applications.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.Http.Headers.html">System.Net.Http.Headers</a></td>
+   <td>
+   Provides support for collections of HTTP headers used by the <a href="http://dotnet.github.io/api/System.Net.Http.html">System.Net.Http namespace</a>.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.NetworkInformation.html">System.Net.NetworkInformation</a></td>
+   <td>
+   Provides access to network traffic data, network address information, and notification of address changes for the local computer. The namespace also contains classes that implement the <a href="http://dotnet.github.io/api/System.Net.NetworkInformation.Ping.html">Ping</a> utility. You can use Ping and related classes to check whether a computer is reachable across the network.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.Security.html">System.Net.Security</a></td>
+   <td>
+   Provides network streams for secure communications between hosts.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Net.Sockets.html">System.Net.Sockets</a></td>
+   <td>
+   Provides a managed implementation of the Windows Sockets (Winsock) interface for developers who need to tightly control access to the network.
```

Comment:
I'm not sure it makes sense to talk about Winsock, since .Net Core is cross-platform.